src/components/SubmitComment.jsx

```diff
@@ -0,0 +1,97 @@
+import { Form, Input, Typography, Button, Space } from 'antd';
+import { useState } from 'react';
+import { postArticleComment } from '../utils/api';
+
+const SubmitComment = ({ article_id, username }) => {
+	const { TextArea } = Input;
+	const { Title } = Typography;
+	const [commentText, setCommentText] = useState('');
+	const [postResponse, setPostResponse] = useState([]);
+
+	const [commentError, setCommentError] = useState('');
+	const [commentSuccess, setCommentSuccess] = useState('');
+
+	const handleInput = (event) => {
+		setCommentText(event.target.value);
+	};
+
```

Comment:
This handle submit seems to show the message 'Posting your comment...' and 'Please enter a comment' when clicked with no message in the box. Please take a look at changing this so you don't have both the error message and the success message showing at the same time 👍